libraries/chain/include/graphene/chain/protocol/nft_ops.hpp

```diff
@@ -0,0 +1,117 @@
+#pragma once
+#include <graphene/chain/protocol/base.hpp>
+#include <graphene/chain/protocol/types.hpp>
+
+namespace graphene { namespace chain {
+
+   struct nft_metadata_create_operation : public base_operation
+   {
+      struct fee_parameters_type { uint64_t fee = GRAPHENE_BLOCKCHAIN_PRECISION; };
+      asset fee;
+
+      account_id_type owner;
+      std::string name;
+      std::string symbol;
+      std::string base_uri;
+      optional<account_id_type> revenue_partner;
+      optional<double> revenue_split;
+      bool isTransferable = false;
+      bool isSellable = true;
+
+      account_id_type fee_payer()const { return owner; }
+      void validate() const;
+   };
+
+   struct nft_metadata_update_operation : public base_operation
+   {
+      struct fee_parameters_type { uint64_t fee = GRAPHENE_BLOCKCHAIN_PRECISION; };
+      asset fee;
+
+      account_id_type owner;
+      nft_metadata_id_type nft_metadata_id;
+      optional<std::string> name;
+      optional<std::string> symbol;
+      optional<std::string> base_uri;
+      optional<account_id_type> revenue_partner;
+      optional<double> revenue_split;
+      optional<bool> isTransferable;
```

Comment:
We should be using underscore (_) as word separator instead of camel case. This field's name should be 'is_transferable' instead of 'isTransferable'.

Reply:
done